src/main/java/com/jdig/model/SorbsBlacklists.java

```diff
@@ -0,0 +1,253 @@
+            "127.0.0.2",
+            "127.0.0.5",
+            "127.0.0.6",
+            "127.0.0.7",
+            "127.0.0.8",
+            "127.0.0.9",
+            "127.0.0.10",
+            "127.0.0.14"
+    ),
+
+    PROBLEMS_DNSBL(
+            "problems.dnsbl.sorbs.net",
+            "Includes http, socks, misc, smtp, new.spam, recent.spam, old.spam, escalations, web, block, zombie zones",
+            "127.0.0.2",
+            "127.0.0.5",
+            "127.0.0.6",
+            "127.0.0.7",
+            "127.0.0.8",
+            "127.0.0.9"
+    ),
+
+    PREEMPT_DNSBL(
+            "preempt.dnsbl.sorbs.net",
+            "Includes dul, noserver zones",
+            "127.0.0.10",
+            "127.0.0.14"
+    ),
+
+    RELAYS_DNSBL(
+            "relays.dnsbl.sorbs.net",
+            "Includes http, socks, misc, smtp zones",
+            "127.0.0.2",
+            "127.0.0.3",
+            "127.0.0.4",
+            "127.0.0.5"
+    ),
+
+    PROXIES_DNSBL(
+            "proxies.dnsbl.sorbs.net",
+            "Includes http, socks, misc zones",
+            "127.0.0.2",
+            "127.0.0.3",
+            "127.0.0.4"
+    ),
+
+    L1_SPEWS(
+            "l1.spews.dnsbl.sorbs.net",
+            "SPEWS Level one listings",
+            "127.0.0.2"
+    ),
+
+    L2_SPEWS(
+            "l2.spews.dnsbl.sorbs.net",
+            "SPEWS Level two listings",
+            "127.0.0.2"
+    ),
+
+    SPEWS_DNSBL(
+            "spews.dnsbl.sorbs.net",
+            "ASPEWS Listings (includes level 1 and level 2)",
+            "127.0.0.2"
+    );
+
+
+    private final String sorbsHostname;
+    private final String description;
+    private final String[] resultCode;
+    private final Set<String> resultCodeSet = new HashSet<String>();
+
+    /**
+     * Creates a SORBS blacklist enum.
+     *
+     * @param sorbsHostname  hostname
+     * @param description    description
+     * @param resultCode     one or more result codes
+     */
+    SorbsBlacklists(
+            String sorbsHostname, String description, String... resultCode) {
+        this.sorbsHostname = sorbsHostname;
+        this.description = description;
+        this.resultCode = resultCode;
+        if (resultCode != null) {
+            Collections.addAll(resultCodeSet, resultCode);
+        }
+    }
+
+    /**
+     * @param ips  one or more IP addresses returned on a lookup to the blacklist
+     *             service
+     * @return     {@code true} if the codes are found on any of the IP addresses
+     *             provided appears on the result codes defined by the
+     *             blacklist.
+     */
+    public boolean isBlacklisted(String ... ips) {
+        if (ips != null) {
+            for (String code : ips) {
+                if (resultCodeSet.contains(code)) {
+                    return true;
+                }
+            }
+        }
+        return false;
+    }
+
```

Comment:
We'll need a public method to get the blacklist url so we can use it at the DnsService class, for instance.